Sybrendj2001/Cinema
# Eg add

- added e.g. by the examples where needed in the admin and customer page
- placed the progress bar in show time different so when admin or other pages make use of the show time it won't show the progress bar

to test
- Go through customer and admin to see if every example is given with e.g.
- See in admin by see movie information if when you see times if there is a progress bar(it should not be there)
